rust/src/instruction/gate.rs

```diff
@@ -0,0 +1,19 @@
+use pyo3::prelude::*;
+
+#[pyclass]
+#[derive(Clone, Debug)]
+pub enum GateModifier {
+    Controlled,
+    Dagger,
+    Forked,
+}
+
+impl From<GateModifier> for quil_rs::instruction::GateModifier {
```

Comment:
To ensure there's no drift between these two, either:

1. Add a `From` in the opposite direction
2. Add a test containing a reversed `match` that will fail to compile if a new variant is added to `quil_rs::GateModifier`

Reply:
Good tip, added the From in the opposite dirction.